### PR TITLE
Cleaned up readme a bit

### PR DESCRIPTION
**Pull request by RichardLitt**
Related to #8

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,20 @@
 # multihash
 
+> hashpipe - pipe iff the hash matches
+
```
Comment (jbenet) [inline README.md]: why is this here?

File: README.md [hunk 2]
```diff
@@ -1,8 +1,20 @@
...
 Multihash is a protocol for differentiating outputs from various well-established cryptographic hash functions, addressing size + encoding considerations.
 
-It is useful to write applications that future-proof their use of hashes, and allow multiple hash functions to coexist. See https://github.com/jbenet/random-ideas/issues/1 for a longer discussion.
+It is useful to write applications that future-proof their use of hashes, and allow multiple hash functions to coexist. See [this GitHub thread](https://github.com/jbenet/random-ideas/issues/1) for a longer discussion.
```
Comment (jbenet) [inline README.md]: 👎 i don't think it helps to hide what the link is.  the words  "this github thread" contain strictly less information than that link. and i do not think that link detracts from the paragraph

File: README.md [hunk 3]
```diff
@@ -1,8 +1,20 @@
...
+
+## Table of Contents
+
+- [Example](#example)
+- [Format](#format)
+- [Implementations:](#implementations)
+- [Table for Multihash v1.0.0-RC (semver)](#table-for-multihash-v100-rc-semver)
+- [Disclaimers](#disclaimers)
+- [Contribute](#contribute)
+- [License](#license)
```
Comment (jbenet) [inline README.md]: Do we need a table of contents? the Readme is pretty short. i'm worried it will be me more trouble to maintain working + updated

Comment (jbenet) [inline README.md]: if it's part of standard-readme, that WFM.

File: README.md [hunk 4]
```diff
@@ -98,4 +110,10 @@ They disagree. :(
 
 Warning: **obviously multihash values bias the first two bytes**. Do not expect them to be uniformly distributed. The entropy size is `len(multihash) - 2`. Skip the first two bytes when using them with bloom filters, etc. Why not _ap_pend instead of _pre_pend? Because when reading a stream of hashes, you can know the length of the hash (from the table).
 
-License: MIT
+## Contribute
+
+PRs and help welcomed. [Take a look at the issues](https://github.com/jbenet/multihash/issues)!
+
+## Licenst
```
Comment (jbenet) [inline README.md]: License

**Comment by jbenet** [comment]
Some comments above

File: README.md (hunk 2 above)
Comment (RichardLitt) [inline README.md]: I disagree; I find parsing a URL in a paragraph of text to be difficult, and I don't think that we should ask the reader to do that. Switching it to "See jbenet/random-ideas#1" with a link would be a good compromise, I think, because that's more readable.

File: README.md (hunk 3 above)
Comment (RichardLitt) [inline README.md]: Its part of the standard. I think it's important, because it saves a lot of time just glancing at the document, and you don't know the length of their output screen. Could be using less or something, with only 30 lines.

**Comment by RichardLitt** [comment]
Fixed comments.

**Comment by diasdavid** [comment]
needs rebase
